sktime/annotation/datagen.py

```diff
@@ -196,6 +196,77 @@ def piecewise_normal(
     return np.concatenate(tuple(segments_data))
 
 
+def piecewise_multinomial(
+    n_trials: int,
+    lengths: npt.ArrayLike,
+    pvals: npt.ArrayLike,
```

Comment:
Here and int he function body you have `pvals` variable name however in the doc string text it's `p_vals`. I guess it makes sense to use `p_vals` since you also chosen `n_trials`. It's a minor thing but consistency often helps.